### PR TITLE
default hostname to InetAddress.getLocalHost.getHostAddress()

### PR DESCRIPTION
**Pull request by sebastianharko**
Default hostname to InetAddress.getLocalHost.getHostAddress() when left empty ""..

[Issue link](https://github.com/akka/akka-cluster-management/issues/11)

### CODE REVIEW

**Comment by typesafehub-validator** [comment]
Hi @sebastianharko,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Lightbend Contributors License Agreement:

[http://www.lightbend.com/contribute/cla](http://www.lightbend.com/contribute/cla)

File: cluster-http/src/main/scala/akka/cluster/http/management/ClusterHttpManagementSettings.scala [hunk 1]
```diff
@@ -3,10 +3,18 @@
...
+
 import com.typesafe.config.Config
 
 final class ClusterHttpManagementSettings(val config: Config) {
   private val cc = config.getConfig("akka.cluster.http.management")
   val ClusterHttpManagementPort = cc.getInt("port")
-  val ClusterHttpManagementHostname = cc.getString("hostname")
+  val ClusterHttpManagementHostname = {
+    val hostname = cc.getString("hostname")
+    if (hostname == "")
```
Comment (2m) [inline cluster-http/src/main/scala/akka/cluster/http/management/ClusterHttpManagementSettings.scala]: Great. Can you also amend the comment section in reference.conf and mention that empty string in hostname has a special meaning?

Comment (sebastianharko) [inline cluster-http/src/main/scala/akka/cluster/http/management/ClusterHttpManagementSettings.scala]: @2m done!

**Comment by 2m** [comment]
Great. Thanks!

**Pull request merged**